programmers/두-정수-사이의-합.js

```diff
@@ -0,0 +1,17 @@
+let answer = 0;
+function solution(a, b) {
+    if (a > b) {
+        for (let i = a; i >= b; i--) {
+            answer += i;
+        }
+    }
+    else if (a < b) {
+        for (let i = a; i <= b; i++) {
+            answer += i;
+        }
+    }
+    else {
+        answer = a;
+    }
+    return answer;
+}
```

Comment:
1. 변수 선언할 때 최대한 함수 내부에서 선언해주세요. 자바스크립트는 전역 변수에 대해서 모든 곳에서 접근이 가능하기 때문에 지양하는게 좋습니다.

2. if...else문은 한 줄로 작성 시에 { } 를 제거할 수 있습니다.

3. 공백이나 들여쓰기는 잘해주셨네요 굿굿
```suggestion
function solution(a, b) {
    let answer = 0;
    
    if (a > b) {
        for (let i = a; i >= b; i--) {
            answer += i;
        }
    } else if (a < b) {
        for (let i = a; i <= b; i++) {
            answer += i;
        }
    } else answer = a;
    
    return answer;
}
```
잘푸셨는데, 최댓값을 구해주는 Math.max( ) 메서드, 최솟값을Math.min( ) 메서드를 이용해서도 풀 수 있습니다.
```js
function solution(a, b) {
    const MIN_NUM = Math.min(a, b);
    const MAX_NUM = Math.max(a, b);
    let answer = 0;

    for (let i = MIN_NUM; i < MAX_NUM + 1; i++) { 
        answer += i;
    }
    
    return answer;
}
```